index.html

```diff
@@ -691,6 +701,8 @@ <h3>
           </li>
           <li>Remove |lock| from |record|.{{[[ActiveLocks]]}}.
           </li>
+          <li>Set |lock|.{{WakeLockSentinel/released}} to `true`.
```

Comment:
This needs to move to being a sub-step of "queue a task" (i.e., this should be step 7.1). The rest all looks good!

Reply:
Do you mean something like:
* Queue a task to run the following steps:
  * Set lock.released to true
  * Fire an event named "release" at lock

Doesn't this mean there'd be a short window between removing `lock` from `[[ActiveLocks]]`/releasing the platform lock and running that task where `released` would still be false and one could even run `WakeLockSentinel.release()` without it short-circuiting?

Reply:
... OTOH, I can also see how it makes sense, as it's the difference between `lock.released` returning `true` immediately after a call to `lock.released()` vs only being reliably `true` after the event is delivered.

Reply:
In the meantime, I've pushed another commit implementing your suggestion.